src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -205,6 +205,25 @@
       "required": [ "message", "locations" ]
     },
 
+    "attachment": {
+      "description": "A file relevant to a tool invocation or to a result.",
+      "type": "object",
+      "properties": {
+
+        "description": {
+          "description": "A message describing the role played by the attachment.",
+          "type": "string"
```

Comment:
The type of the description property should be a `message` object, so instead of
```
"type": "string"
```
you need to write
```
"$ref": "#/definitions/message"
```